go/vt/vtadmin/cluster/config.go

```diff
@@ -237,7 +241,7 @@ func (cfg *Config) MarshalJSON() ([]byte, error) {
 // config. Neither the caller or the argument are modified in any way.
 func (cfg Config) Merge(override Config) Config {
 	merged := Config{
-		ID:                          cfg.ID,
+		ID:                          formatID(cfg.ID),
```

Comment:
can we do this further up the call stack (when we parse a config) so we don't need to do this operation every time we merge two configs?

Reply:
Yes! 😄